src/MycilaESPConnect.h

```diff
@@ -220,6 +220,27 @@ namespace Mycila {
       // when using auto-load and save of configuration, this method can clear saved states.
       void clearConfiguration();
 
+      // Get the scheduled restart time
+      uint32_t getrestartRequestTime() const { return _restartRequestTime; }
+
+      // Set the scheduled restart time
+      void setRestartRequestTime(uint32_t time) {
+        _restartRequestTime = time;
+        if (time == 0) {
+          _restartDelay = 0;
+        }
+      }
```

Comment:
to remove: this variable is just there to record the restart request time. This is not a user-facing stuff.